server/routes/route.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import partyRouter from './parties';
+
+const router = express.Router();
+
+router.use('/parties', partyRouter);
+
+export default router;
```

Comment:
Don't you think it's better you the file name is index.js instead of route.js considering that will be entry file in routes folder?